### ISSUE TITLE
Error with 2.6.8 version

### ISSUE THREAD

**Issue by scare82** [seriousme/fastify-openapi-glue#330]
Hi

I've a problem with 2.6.8 version of this plugin. With 2.6.7 my yaml swagger specification file works fine, but with 2.6.8 update i've this error: 
ENOENT: no such file or directory, open '/D:/pode/ts-employee-backend-write/node_modules/@seriousme/openapi-schema-validator/schemas/v2.0/schema.json'
Error: 'specification' parameter must contain a valid version 2.0 or 3.0.x or 3.1.x specification
    at parser.parse (D:\pode\ts-employee-backend-write\node_modules\fastify-openapi-glue\lib\parser.js:54:13)
    at async fastifyOpenapiGlue (D:\pode\ts-employee-backend-write\node_modules\fastify-openapi-glue\index.js:114:18)

My code with openapi-glue is very simple:

```
import FastifyOpenApiGlue from 'fastify-openapi-glue';
.
.
.
.
    const openapiGlueOptions = {
        specification: path.join(__dirname, 'api/swagger.yaml'),
        service: new WriteController()
    };
    const app = Fastify({
        bodyLimit: 524288,
        maxParamLength: 200
    });
    app.register(FastifyOpenApiGlue, openapiGlueOptions);
```

This is an example that works with 2.6.7 but brokes with 2.6.8

```
swagger: "2.0"
info:
    description: |
        TODO

    version: "5.0.1"
    title: WIP
    termsOfService: http://swagger.io/terms/
    contact:
        email: xxx@xxxxx.com
    license:
        name: Apache 2.0
        url: http://www.apache.org/licenses/LICENSE-2.0.html
externalDocs:
    description: Find out more about Swagger
    url: http://swagger.io
basePath: /
schemes:
    - https
    - http
produces:
    - application/json

paths:
    /docs/isReady:
        get:
            tags:
                - Probes
            summary: isReady (for k8s readiness probe)
            description: isReady (for k8s readiness probe)
            operationId: isReady
            responses:
                200:
                    description: isReady
                500:
                    description: ko
```

Sorry for my poor English

Filippo

**Comment by seriousme** [comment]
Hi Filippo,

thanks for asking !

The  error seems to indicate that the file: '/D:/pode/ts-employee-backend-write/node_modules/@seriousme/openapi-schema-validator/schemas/v2.0/schema.json' does not exist.
Can you please check the output of the following command:
```bash
npm ls @seriousme/openapi-schema-validator
```
This should return:
```bash
fastify-openapi-glue@2.6.8 /home/hansklunder/github/fastify-openapi-glue
└── @seriousme/openapi-schema-validator@2.0.0 
```
If it does not give this answer then you might need to run `npm install` again.

Kind regards,
Hans

**Comment by scare82** [comment]
Hi Hans and thanks for reply.

` npm ls @seriousme/openapi-schema-validator`

return

```
fastify-openapi-glue@2.6.8
  └── @seriousme/openapi-schema-validator@2.0.1

```

I've try with `yarn remove fastify-openapi-glue` and `yarn add fastify-openapi-glue` but the error remains

Fastify-openapi-glue 2.6.7 depends of 1.7.1 openapi-schema-validator, but in 2.6.8 there is 2.0.1.
openapi-schema-validator 2.0.0 migrated to ESM module, my project is still commonjs. May be the problem?

**Comment by seriousme** [comment]
Hi Filippo,

CommonJS vs ESM is not the problem as `@seriousme/openapi-schema-validator` is loaded ;-)
This can be derived from the error, it can't open: "@seriousme/openapi-schema-validator/schemas/v2.0/schema.json"  and this is only opened from "@seriousme/openapi-schema-validator".

My suggestion would be to check if `/D:/pode/ts-employee-backend-write/node_modules/@seriousme/openapi-schema-validator/schemas/v2.0/schema.json` exists. 

If it exists then there must be something wromg with the file system access policy.

If it does not exist then something went wrong with the installation. My suggestion would then be to remove the `/D:/pode/ts-employee-backend-write/node_modules/` folder and do a fresh `yarn install`.

Hope this helps !

Kind regards,
Hans

**Comment by scare82** [comment]
Hi Hans and sorry delay for reply.

 Finally i found the problem but not solution. The problem is Windows. In Linux works but in Windows your script try to open /D:/pode/ts-employee-backend-write/node_modules/@seriousme/openapi-schema-validator/schemas/v2.0/schema.json. 

The problem is localFile function of openapi-schema-validator. A possible correction that works in linux and windows (from https://nodejs.org/api/url.html#urlfileurltopathurl):

```
import { fileURLToPath } from 'url';


function localFile(fileName) {
    return fileURLToPath('file://' + new URL(fileName, import.meta.url).pathname);
}

```

Once again, Sorry for my poor english.

Your fastify plugin is very useful

**Comment by seriousme** [comment]
Hi Filippo,

thanks for finding both the problem and the solution !
I have released version 2.0.2 of @seriousme/openapi-schema-validator with the fix in it.

I also released fastify-openapi-glue as version 2.6.9 that includes @seriousme/openapi-schema-validator version 2.0.2

Your english is good enough for me ;-)

Thanks again for helping !
Hans
